XamarinCommunityToolkit/Converters/IntToBoolConverter.shared.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Converters
+{
+    public class IntToBoolConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+            => (int)value != 0;
```

Comment:
@sthewissen added throw new ArgumentException when the value being passed is not of the proposed type. So maybe keep that same pattern and check if the value passed is an int in this case.